### PR TITLE
feat: Customer section added 

### PR DESCRIPTION
**Pull request by AdrianAlzamoraG**
Features:
- assets folder added, includes css and the images of the corresponding section.
- main.js file updated
- index file updated

### CODE REVIEW

**Review by amenes12 — COMMENTED** [review]
LGTM, great job!

File: index.html [hunk 1]
```diff
@@ -59,10 +62,75 @@
...
+      </div>
+    </div>
+  </div>
+
   <!-- Scripts always are the last items before closing the body tag -->
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
     integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
     crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Swiper/8.1.3/swiper-bundle.min.js" integrity="sha512-dLuSIV553xT28AiFViM1ZQdOlyts81E3xiQvfiltddvQ0RViD99nY8sNlRx5o/9J18BYoRQV+PHo9dGWRv3vjA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
```
Comment (dalbitresb12) [inline index.html]: Can you remove this blank line?

File: public/assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+@import url(./main.css);
```
Comment (dalbitresb12) [inline public/assets/css/styles.css]: Can you move all the contents of this file back to `main.css` and remove this one? Please also move the `/public/css` folder back to the root and remove the `/public` folder.

File: index.html [hunk 3]
```diff
@@ -14,7 +14,10 @@
   <!-- CSS always are the last items before closing the head tag -->
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://unpkg.com/swiper@8/swiper-bundle.min.css"/>
   <link href="styles/main.css" rel="stylesheet">
+  <link href="./public/assets/css/styles.css" rel="stylesheet">
+
```
Comment (dalbitresb12) [inline index.html]: Remove this blank line

File: index.html [hunk 4]
```diff
@@ -14,7 +14,10 @@
   <!-- CSS always are the last items before closing the head tag -->
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://unpkg.com/swiper@8/swiper-bundle.min.css"/>
   <link href="styles/main.css" rel="stylesheet">
+  <link href="./public/assets/css/styles.css" rel="stylesheet">
```
Comment (dalbitresb12) [inline index.html]: We should only have the `styles/main.css` file, we don't need this one.

File: index.html [hunk 5]
```diff
@@ -59,10 +62,75 @@
     </div>
   </nav>
 
+  <!--Our Customers-->
+  <div class="slider-1">
```
Comment (dalbitresb12) [inline index.html]: Wrap your component in a `<section>` tag with an id.

File: scripts/main.js [hunk 6]
```diff
@@ -1 +1,38 @@
 console.log("[main.js] Test main.js");
+setTimeout(function () {
+
+    /* Image Slider - Swiper */
+    var imageSlider = new Swiper(".image-slider", {
```
Comment (dalbitresb12) [inline scripts/main.js]: Avoid using `var`. Prefer `const` (or `let` if you need to reassign its value).

```suggestion
    const imageSlider = new Swiper(".image-slider", {
```

File: scripts/main.js [hunk 7]
```diff
@@ -1 +1,38 @@
 console.log("[main.js] Test main.js");
+setTimeout(function () {
```
Comment (dalbitresb12) [inline scripts/main.js]: Waiting for the [`load` event](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) to fire would work better here.

```suggestion
window.addEventListener("load", () => {
```

**Review by dalbitresb12 — CHANGES_REQUESTED** [review]
Just minor changes.

File: index.html [hunk 8]
```diff
@@ -59,10 +62,75 @@
...
+          <hr class="section-divider" />
+
+          <div class="slider-container">
+            <div class="swiper-container image-slider">
+              <div class="swiper-wrapper">
+                <div class="swiper-slide">
+                  <img
+                          class="img-fluid"
+                          src="./public/assets/images/customer-logo-1.png"
+                          alt="alternative"
```
Comment (dalbitresb12) [inline index.html]: Forgot to tell you to set the `alt` attributes properly (they should include a description of the picture).

Comment (dalbitresb12) [reply index.html]: This applies to all `` tags you add.

File: styles/main.css [hunk 9]
```diff
@@ -2,3 +2,97 @@
...
+  vertical-align: middle;
+}
+
+
+
+/* Min-width 1200px */
+@media (min-width: 1200px) {
+
+  /* General Styles */
+  .container {
```
Comment (dalbitresb12) [inline styles/main.css]: Same thing about not overriding Bootstrap classes.

File: index.html (hunk 3 above)
Comment (dalbitresb12) [reply index.html]: You forgot about this.

File: styles/main.css [hunk 10]
```diff
@@ -2,3 +2,97 @@
 :root {
   --bs-font-sans-serif: 'Roboto', system-ui, -apple-system, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 }
+
+
+div {
+  display: block;
+}
```
Comment (dalbitresb12) [inline styles/main.css]: This is by default in all browsers.
```suggestion
```

File: styles/main.css [hunk 11]
```diff
@@ -2,3 +2,97 @@
...
+
+/********************/
+/*    Customers     */
+/********************/
+
+.blue {
```
Comment (dalbitresb12) [inline styles/main.css]: Would be better if you set the name like this:

```suggestion
.text-blue {
```

File: styles/main.css [hunk 12]
```diff
@@ -2,3 +2,97 @@
...
+  color: #00cec9;
+}
+.img-fluid {
+  max-width: 100%;
+  height: auto;
+  margin-left: auto;
+  display: block;
+}
```
Comment (dalbitresb12) [inline styles/main.css]: Bootstrap [already has this](https://getbootstrap.com/docs/5.0/content/images/#responsive-images).

```suggestion
```

File: styles/main.css [hunk 13]
```diff
@@ -2,3 +2,97 @@
...
+
+.slider-1 {
+  padding-top: 2rem;
+  padding-bottom: 3.75rem;
+  background: linear-gradient(rgb(238, 238, 238), rgb(144, 173, 198));
```
Comment (dalbitresb12) [inline styles/main.css]: I think this looks better without the gradient, but I don't know about everyone else.

@Future-Leaders-SI653/development what do you guys think?

```suggestion
```

File: styles/main.css [hunk 14]
```diff
@@ -2,3 +2,97 @@
...
+
+.row {
+  --bs-gutter-x: 1.5rem;
+  --bs-gutter-y: 0;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: calc(var(--bs-gutter-y) * -1);
+  margin-right: calc(var(--bs-gutter-x)/ -2);
+  margin-left: calc(var(--bs-gutter-x)/ -2);
+}
```
Comment (dalbitresb12) [inline styles/main.css]: Try not to override Bootstrap classes and variables, just apply the changes for your components. You can also change the `margin` properties to the [summarized syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax)

```suggestion
.some-class-name {
  display: flex;
  flex-wrap: wrap;
  margin: 0 -0.75rem;
}
```

File: index.html [hunk 15]
```diff
@@ -58,11 +61,73 @@
       </div>
     </div>
   </nav>
+  <!--Our Customers-->
+  <section class="slider-1" id="slider-section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <h4>
+            Trusted by over <span class="blue">1500</span> customers worldwide
```
Comment (dalbitresb12) [inline index.html]: If you change the class name, don't forget to change it here as well.

```suggestion
            Trusted by over <span class="text-blue">1500</span> customers worldwide
```

**Review by dalbitresb12 — CHANGES_REQUESTED** [review]
A little bit more of changes, after this I think it's ready for merging.

File: styles/main.css (hunk 9 above)
Comment (AdrianAlzamoraG) [reply styles/main.css]: Reviewed and corrected

File: styles/main.css (hunk 10 above)
Comment (AdrianAlzamoraG) [reply styles/main.css]: Reviewed and corrected

File: styles/main.css (hunk 11 above)
Comment (AdrianAlzamoraG) [reply styles/main.css]: Reviewed and corrected

File: styles/main.css (hunk 12 above)
Comment (AdrianAlzamoraG) [reply styles/main.css]: Reviewed and corrected

File: styles/main.css (hunk 13 above)
Comment (AdrianAlzamoraG) [reply styles/main.css]: Ready!

File: styles/main.css [hunk 16]
```diff
@@ -1,4 +1,62 @@
-/* Override Bootstrap theme */
-:root {
-  --bs-font-sans-serif: 'Roboto', system-ui, -apple-system, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```
Comment (dalbitresb12) [inline styles/main.css:3]: Add this back please.

**Review by dalbitresb12 — CHANGES_REQUESTED** [review]
Everything else looks okay, just this.

**Review by dalbitresb12 — APPROVED** [review]
LGTM

**Comment by dalbitresb12** [comment]
Just solved a couple of conflicts and code formatting. Also moved images to the correct folder.

**Pull request merged**